ui/page/verify_message_page.go

```diff
@@ -170,11 +173,11 @@ func (pg *VerifyMessagePage) Handle() {
 		}
 	}
 
-	if pg.verifyButton.Button.Clicked() || handleSubmitEvent(pg.addressEditor.Editor, pg.messageEditor.Editor, pg.signatureEditor.Editor) {
+	if pg.verifyButton.Button.Clicked() || handleSubmitEvent(pg.addressEditor.Editor, pg.signatureEditor.Editor, pg.messageEditor.Editor) {
 		if pg.validateAllInputs() {
 			pg.verifyMessage.Text = ""
 			pg.verifyMessageStatus = nil
-			valid, err := pg.WL.MultiWallet.VerifyMessage(pg.addressEditor.Editor.Text(), pg.messageEditor.Editor.Text(), pg.signatureEditor.Editor.Text())
+			valid, err := pg.WL.MultiWallet.VerifyMessage(pg.addressEditor.Editor.Text(), pg.signatureEditor.Editor.Text(), pg.messageEditor.Editor.Text())
```

Comment:
kindly revert this. the dcrlibwallet Verify message takes the message before the signature.
`func (mw *MultiWallet) VerifyMessage(address string, message string, signatureBase64 string) (bool, error)`